CS/switcher/switcher.sln

```diff
@@ -12,12 +12,6 @@ Global
 		Release|Any CPU = Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{84CEBC50-4513-490B-B138-2994138270BC}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
```

Comment:
Are this changes needed?

Reply:
i needed to make a whole new project because rider broke everything after renaming the root namespace (lol)
i'll revert the .sln changes since they're unnecessary